src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/InvitationTests.cs

```diff
@@ -37,6 +37,7 @@ public class InvitationTests
         private const string Title = "Title A";
         private const string Title2 = "Title B";
         private const string Description = "Description A";
+        private const string System = "12";
```

Comment:
In real life, will it be possible to have McPkg under a System (not SubSystem)? Should McPkg constructor validate on a legal System-string (must have | and be 3 chars or more?)

Reply:
No, this system is not valid, the system will always include a '|'. Will ad this check

Reply:
The constructor should then throw Exception

Reply:
Not ArgumentException ?

Reply:
ArgumentException